actions/utils.py

```diff
@@ -181,11 +184,15 @@ def get_interface():
     """
     Chooses an interface on the machine to use for socket testing.
     """
-    ifaces = netifaces.interfaces()
-    for iface in ifaces:
-        if "lo" in iface:
-            continue
-        info = netifaces.ifaddresses(iface)
-        # Filter for IPv4 addresses
-        if netifaces.AF_INET in info:
-            return iface
+    if os.name == 'nt':
```

Comment:
your branching is funky - see my comments on the windows code review (#14) for the rest of this